### ISSUE TITLE
Build issue on HEAD

### ISSUE THREAD

**Issue by jsmaniac** [jbclements/RSound#30]
With the latest nightly (minimal racket, with all "base" packages updated yesterday), I had the following error:

```
> racket                                                                                                                                                                                             phc  ~/phc/racket-packages/phc-thesis
Welcome to Racket v6.8.0.2.
> (require rsound)
; /home/georges/.racket/snapshot/pkgs/rsound/rsound/util.rkt:19:9: module:
;   identifier already imported from: "common.rkt"
;   at: positive-integer?
;   in: math/base
; [,bt for context]
```

I changed the `math/base` on line 19 in `util.rkt` to the following, and it seems to work fine:

```
         racket/require
         (subtract-in math/base "common.rkt")
```

It might be an issue with stale bytecode files on my machine, though, so it's probably best to check that you can reproduce the issue before looking into this.

**Comment by jsmaniac** [comment]
I also had to change the require for `racket/gui` in `draw.rkt` to the following:

```
         racket/require
         (subtract-in racket/gui "common.rkt")
```

**Comment by benoid** [comment]
Do you know why this is suddenly happening?  I can not replicate the error in racket 6.7, and I am not sure I fully comprehend the underlying cause of the problem.

**Comment by jbclements** [comment]
Yes, this is a super-minor issue. The problem is that rsound defines a function called positive-integer?, and the math library now provides a function with that name, causing a conflict. It should be a very minor fix. I've been meaning to push out the 44.1/48K pref-switch version as well. Let me see if I can do both of these today.

**Comment by benoid** [comment]
Oh alright, thanks!

**Comment by jbclements** [comment]
this bug closed (back in march) by 206c18ffae8e086875ad, thanks for the report!
